Bottom: 5f907a3bca2067182a3c6b4740ec325d6ad128b4
Top:    8acd928caaa38d1cba1d1d9a85957184fa78d7b5
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-09-27 10:46:17 -0500

Add Changepassword.php page to phpldapadmin



---
diff --git a/docker/docker-phpldapadmin/Dockerfile b/docker/docker-phpldapadmin/Dockerfile
index 4544e647d..249051d65 100644
--- a/docker/docker-phpldapadmin/Dockerfile
+++ b/docker/docker-phpldapadmin/Dockerfile
@@ -1,3 +1,3 @@
 FROM osixia/phpldapadmin:latest
 
-COPY changepassword.php /var/www/changepassword.php
+COPY changepassword.php /var/www/phpldapadmin_bootstrap/htdocs/changepassword.php
